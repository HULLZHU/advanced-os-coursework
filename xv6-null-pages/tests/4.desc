test bad arguments for munprotect system call
    * len <=0
    * address not in page range (PGSIZE)
