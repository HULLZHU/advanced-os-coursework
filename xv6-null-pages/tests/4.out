XV6_TEST_OUTPUT: Return value of munprotect with len 0: -1
XV6_TEST_OUTPUT: Return value of munprotect with len -1: -1
XV6_TEST_OUTPUT: Return value of munprotect with unaligned Page Address: -1
